nav/right.guard.ts

```diff
@@ -82,3 +82,66 @@ export class RightGuard  {
         return url;
     }
 }
+
+export const RightGuard: CanActivateFn = (route: ActivatedRouteSnapshot, state: RouterStateSnapshot): boolean => inject(RightGuardService).canActivate(route, state);
+
+// @Injectable()
+// export class RightGuard  {
+
+//     constructor(private readonly authService: AuthService, private readonly router: Router) {
+//     }

```

Comment:
if you made your job right and the transformed guards work as before, we don't want to merge the old out-commented code.